OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,24 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
+    /** Retrieve the report.                                                  */
     const TimeSeriesTable_<ValueT>& getReport() const {
         return _outputTable;
     }
 
+    /** Clear the report. This can be used for example in loops where iteration
+    is intended to create a fresh report.                                     */
```

Comment:
+1 for the example, but I think the example can be a little clearer ("performing multiple simulations starting from the same initial time"). I think the comment should mention that the input will still be connected; that the channels set up are still wired up.